unified-runtime/scripts/core/device.yml

```diff
@@ -462,6 +462,8 @@ etors:
       desc: "[int32_t][optional-query] return max power limit in milliwatts."
     - name: BFLOAT16_CONVERSIONS_NATIVE
       desc: "[$x_bool_t] support for native bfloat16 conversions"
+    - name: KERNEL_LAUNCH_PROPERTIES_SUPPORT
+      desc: "[$x_kernel_launch_properties_support_flags_t] Bitfield of supported kernel launch properties."
```

Comment:
I thought the `_SUPPORT` suffix on these was for queries that return bool rather than flags, dropping it would also be a chunk less typing.

Reply:
dropped `support` from the type name and ended up going with KERNEL_LAUNCH_CAPABILITIES for the query name, CAPABILITIES being a common bitfield query trope

Reply:
makes sense 👍